nextflow_schema.json

```diff
@@ -1457,6 +1462,37 @@
             },
             "fa_icon": "fas fa-angle-double-right"
         },
+        "cazyme_dbcan": {
+            "title": "dbCAN",
+            "type": "object",
+            "description": "Carbohydrate-active enzyme annotation based on pre-defined HMM models.\n\nFor more information check the dbCAN [documentation](https://run-dbcan.readthedocs.io/en/latest)",
+            "default": "",
+            "properties": {
+                "cazyme_skip_dbcan": {
+                    "type": "boolean",
+                    "description": "Skip dbCAN during the CAZyme screening.",
+                    "fa_icon": "fas fa-ban"
+                },
+                "dbcan_skip_cgc": {
+                    "type": "boolean",
+                    "description": "Skip CGC during the dbCAN screening.",
+                    "fa_icon": "fas fa-ban"
+                },
+                "dbcan_skip_substrate": {
+                    "type": "boolean",
+                    "description": "Skip substrate during the dbCAN screening.",
+                    "fa_icon": "fas fa-ban"
+                },
+                "dbcan_gff_type": {
+                    "type": "string",
+                    "default": "prodigal",
+                    "description": "GFF type used to annotate CAZymes in the input GFF file.",
+                    "help_text": "This flag specifies the type of GFF file that is used to annotate CAZymes in the input GFF file. It is used to ensure that the correct gene annotations are used for the CAZyme screening. For more information check the dbCAN [documentation](https://run-dbcan.readthedocs.io/en/latest/user_guide/CGC_information_generation.html#input-gff-file-types).",
```

Comment:
I think this information shoiuld be pushed to the samplesheet as another optional column - even if it would weird, maybe someone would have different GFF types for each sample?